composite-actions/frontend-generic/upload-source-maps/README.md

```diff
@@ -11,6 +11,7 @@ This is a shared action for a frontend application: uploading source maps to Ela
     SERVICE_NAME: "your-service-name"
     SERVICE_VERSION: "your-service-version"
     BUILD_DIR: "dist"
+    PREFIX: "your-prefix"
```

Comment:
prefix should have trailing "/" and the prefix "/" .i.e "/", "/recon/". Also update the code to take / from user input . remove the hardcoded "/"